Adding details to README for ACA data analysis. (#10)
ConnectedSystems/GBR-Bathy10m: README.md

[aca_data]
ALLEN_ATLAS_DIR = "path to Allen Atlas data"  # location of Allen Atlas datasets

[wave_data]
WAVE_DATA_DIR = "path to wave data" # location of wave NetCDF datasets
```

### Data layout

Expected data directory layout:

Separate directories are used for GBRMPA (MPA), Allen Coral Atlas (ACA) and wave datasets.
Sub-directory names should be consistent and match.

MPA_DATA_DIR : contains raster data at whole-GBR and GBRMPA-management-region scales.
- `zones` holds GBRMPA zone layers in geojson format.
- `features` holds the GBRMPA GBR-wide feature set.

ACA_DATA_DIR : contains raster and vector data at whole-GBR scale.

> WAVE_DATA_DIR : contains wave NetCDF data at GBRMPA-management-region scale.

Reviewer: Something like this?

```suggestion
WAVE_DATA_DIR : contains wave data in NetCDF format at the scale of GBRMPA management regions.
```

│   ├───FarNorthern
│   ├───Mackay-Capricorn
│   └───Townsville-Whitsunday
└───zones

ACA_DATA_DIR
├───Bathymetry---composite-depth
│   └───Raster data
├───Benthic-Map
│   └───Vector data
├───boundary
│   └───Vector data
├───Geomorphic-Map
│   └───Vector data
├───Reef-Extent
│   └───Vector data
├───Turbidity-Q3-2023
│   └───Raster data

> 

Reviewer: ```suggestion
│   └───Vector data
└───Turbidity-Q3-2023
    └───Raster data

```

WAVE_DATA_DIR
├───Hs
│   ├───Cairns-Cooktown
│   ├───FarNorthern
│   ├───Mackay-Capricorn
│   └───Townsville-Whitsunday

> ├───Tp
> │   ├───Cairns-Cooktown
> │   ├───FarNorthern
> │   ├───Mackay-Capricorn
> │   └───Townsville-Whitsunday

Reviewer: ```suggestion
└───Tp
    ├───Cairns-Cooktown
    ├───FarNorthern
    ├───Mackay-Capricorn
    └───Townsville-Whitsunday
```

https://doi.org/10.48610/8246441
https://espace.library.uq.edu.au/view/UQ:8246441

> ### ACA data
> https://www.allencoralatlas.org/

Reviewer: Spacing is needed

```suggestion
### ACA data

https://www.allencoralatlas.org/
```

Often this might be the only data available.

#### Resolution

`MPA raster data` (Bathymetry, Benthic, Geomorphic, Slope, Waves) : 10 x 10m pixel size

> `ACA raster data` (Bathymetry, Turbidity, Waves) : 10 x 10m pixel size

Reviewer 2: Separate waves as standalone dataset, netCDF format
